api/src/main/java/org/openmrs/util/databasechange/ConceptValidatorChangeSet.java

```diff
@@ -131,15 +131,16 @@ private void validateAndCleanUpConcepts(JdbcConnection connection) {
 			List<ConceptName> namesWithNoLocale = null;
 			
 			//for each locale
-			for (Locale conceptNameLocale : locales) {
```

Comment:
The diff will not show it because you did not change it. But your changes seem to make it irrelevant. If you want to see it, take a look at the actual source file https://github.com/meinsen/openmrs-core/blob/26cf38871f542c57fd2ef3c81aa6a7956b801fa1/api/src/main/java/org/openmrs/util/databasechange/ConceptValidatorChangeSet.java#L130